.eslintrc

```diff
@@ -38,7 +38,8 @@
                 "args": "none"
             }
         ],
-        "comma-dangle": 0,
+        "no-console": 1,                      // 1=warn
+        "comma-dangle": 0,                    // 0=off
```

Comment:
I am not updating LINT.md in this pull request. I would love that contribution thought, for consistency.